### PR TITLE
fix(pinned-cert) correct api base url

### PR DESCRIPTION
**Pull request by jeffyongtaotang**
- Corrects API Base URL
- Pin cert with koko `/v1/dp-client-certificates` endpoint

### CODE REVIEW

File: konnect-runtime-setup.sh [hunk 1]
```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env bash
+KONNECT_GLOBAL_API_URL=${KONNECT_GLOBAL_API_URL:-"https://global.api.konghq.com"}
+KONNECT_API_URL=${KONNECT_API_URL:-"https://us.api.konghq.com"}
```
Comment (jeffyongtaotang) [inline konnect-runtime-setup.sh:3]: from the customer script, to support env var as arguments.

**Review by Farwhyn — APPROVED** [review]
LGTM!

I'm actually thinking if this should be its own branch instead. The reason is that all the existing cert versions require the UI to pass in the param values, whereas here we are modifying the script ourselves.

**Comment by jeffyongtaotang** [comment]
@Farwhyn did you functional test on ur end. It's work for me, try to make sure it also working on ur machine.

**Comment by Farwhyn** [comment]
@jeffyongtaotang verified your script is working, successfully created runtime instance for gateway version 2.7.0.0 on Cloud02 Prod
image

**Pull request closed without merging**